galaxy.yml

```diff
@@ -0,0 +1,22 @@
+namespace: ibm_spectrum_scale
+name: install_infra
+version: v0.0.0
```

Comment:
Any idea how this version can be non static.... meaning, could this piggy back on the git tags/releases?

Reply:
I don't think it is possible, the closest you can get is some pre-submit hooks - but that only applies if you are making the tag on the command line I believe.